src/vs/platform/clipboard/browser/clipboardService.ts

```diff
@@ -13,6 +13,10 @@ export class BrowserClipboardService implements IClipboardService {
 
 	private readonly mapTextToType = new Map<string, string>(); // unsupported in web (only in-memory)
 
+	async write(data: any, type?: string): Promise<void> {
```

Comment:
This shouldn't be `any` type though - you should create a new type 

```
export type ClipboardData = {
{
  text: string;
  html?: string;
  rtf?: string;
  bookmark?: string;
}
```

and use that instead

Reply:
Sure, makes sense. I will add it to clipboardService.ts